manifests/v1alpha2/katib/pv/pv.yaml

```diff
@@ -12,4 +12,4 @@ spec:
   accessModes:
     - ReadWriteOnce
   hostPath:
-    path: /tmp/katib
+    path: /data/katib
```

Comment:
/cc @hougangliu 

/lgtm

Reply:
In the initial version, I use "/data/katib" here. But test cannot pass with error saying "there is no dir /data in host path"-like. I have no idea why it occurs since in v1alpha1, it is "/data/katib" and everything goes well.
LGTM if the test says "LGTM", too :)

Reply:
Maybe we should create the dir in the host?

Reply:
No idea. Let's wait the result

Reply:
Since we are using hostPath, we need to create this folder in the test cluster for v1alpha2. @gaocegege was right.

Reply:
I didn't find where we create "/data" in v1alpha1 test. that's weird